src/requested-data/requested-data-state.ts

```diff
@@ -1,4 +1,5 @@
 import { ElectionType, Votes, Metrics, Parameters } from "./requested-data-models";
+import { Dictionary } from "utilities/dictionary";
```

Comment:
Codacy found an issue: [Module 'utilities' is not listed as dependency in package.json](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5947793)